SeaPublicWebsite/ExternalServices/FileRepositories/VcapServiceFactory.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+
+namespace SeaPublicWebsite.ExternalServices.FileRepositories
+{
+    // TODO: SEABETA-193 Remove this file
+    public class VcapServiceFactory

```

Comment:
This is a mixture of functions pulled from the old Global.cs and Config.cs as they are only needed here now, and we won't even need them here once we move over to a database